src/job_recur.jl

```diff
@@ -58,9 +58,9 @@ Step values can be used in conjunction with ranges. Following a range with `/<nu
 
 `Vector` works like lists mentioned above. For example, `[1,2,5,9]` is equivalent to `"1,2,5,9"`.
 
-## When an argument is a `UInt64`:
+## When an argument is a `UInt`:
 
-`UInt64` is the internal type of `Cron` fileds. All the previous types will be converted to a `UInt64` bit array. The start index of the bit array is 0. Bits outside of the allowed values (see the table above) are ignored.
+`UInt` is the internal type of `Cron` fileds. All the previous types will be converted to a `UInt` bit array. The start index of the bit array is 0. Bits outside of the allowed values (see the table above) are ignored.
 """
 function Cron(; 
     second = 0x0000000000000001, 
```

Comment:
Will this work on 32bit system?
Should the internal structure not be converted to UInt32 too?